_includes/accordion.html

```diff
@@ -15,6 +15,6 @@ <h4 class="usa-accordion__heading" id="{{ include.id }}">
       {{ include.title }}
     </button>
 </h4>
-<div id="{{ include.accordion_id }}" class="usa-accordion__content usa-prose">
+<div id="{{ include.accordion_id }}" class="usa-accordion__content usa-prose" hidden="until-found">
```

Comment:
Is this going to fix the issue with auto expanding the accordions? I made a ticket for it so I can mark it as done / assign it to you